step.sh

```diff
@@ -55,7 +56,6 @@ FILE_SIZE=`jq -r '.result.UploadFileRsp.fileInfoList[0].size' uploadfile.json`
 
```

Comment:
`apk_path` should be renamed to `file_path` also on line 52

![obraz](https://user-images.githubusercontent.com/4972310/88154937-badb6800-cc07-11ea-967b-328840f9f0c3.png)

Reply:
done